src/Swarm/Game/Scenario/Objective/Presentation/Render.hs

```diff
@@ -37,20 +38,26 @@ renderGoalsDisplay gd =
       else goalElaboration
  where
   lw = _listWidget gd
+  fr = _focus gd
   leftSide =
     hLimitPercent 30 $
       vBox
         [ hCenter $ str "Goals"
         , padAll 1 $
             vLimit 10 $
-              BL.renderList (const drawGoalListItem) True lw
+              withFocusRing fr (BL.renderList drawGoalListItem) lw
         ]
-  goalsCount = sum . M.elems . M.map NE.length . goals $ gd ^. goalsContent
+
+  -- Adds very subtle coloring to indicate focus switch
+  highlightIfFocused = case focusGetCurrent fr of
+    Just (GoalWidgets GoalSummary) -> withAttr lightCyanAttr
```

Comment:
I think the light cyan is nice to indicate focus when there's an objective list and a goal description, but I find it annoying that all the goals are now colored light cyan, even when there's only a single objective.

Reply:
fixed